CoEpi/ui/launch screen/CoEpiLaunchScreen.storyboard

```diff
@@ -2,7 +2,6 @@
 <document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15702" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="01J-lp-oVM">
```

Comment:
The splash image looks a bit pixelated. Does it have the correct scale?

Reply:
I messed with this for a while, and could not produce better results. Will look into it later.

Reply:
resolved